### PR TITLE
Add chart version change hints

### PR DESCRIPTION
**Pull request by sophokles73**
Based on the discussion on Gitter I think that it seems reasonable to explicitly describe some circumstances that would/should require a change of a chart's version. Some of the more obvious
cases are documented in the contribution guide. These can serve as a basis for further additions and or changes.

This PR is supposed to serve as a straw man for discussing the subject and coming to an agreement on at least a few basic rules.

### CODE REVIEW

**Comment by ctron** [comment]
I am not sure if we should add and enforce this on a global level. I absolutely understand the reason for semantic versioning, and this might be a good guideline. But I also think that we should leave it up to the projects to make that decision on their own.

**Comment by axdotl** [comment]
What do you think of adding a link to Helm [best practices on versioning](https://helm.sh/docs/topics/chart_best_practices/conventions/#version-numbers)? And maybe a short explanation how semver should be/ is done in the package project.

**Comment by sophokles73** [comment]
I had hoped to find some useful information there as well. However, I found the generic SemVer description to be rather confusing than helpful so I thought it might be a good idea to explain the generic concept by means of some specific examples from our problem domain.
When I asked about how semver would need to be applied to Helm chart versions last week on our Gitter channel, nobody seemed to have an answer so I assumed that others would have similar problems of understanding as me ...

**Comment by sophokles73** [comment]
Adding a link to the best practices on version still makes sense, of course, FMPOV.

File: homepage/contribute.md [hunk 1]
```diff
@@ -122,6 +122,20 @@ At least one maintainer needs to explicitly approve the PR before it can be merg
...
+of changes that should be reflected by a corresponding increase of the patch, minor or major
+version of the chart.
+
+| Type of change | Requires increase of |
+| -------------- | -------------------- |
+| Chart introduces a property for explicitly configuring an already existing fixed value | patch version |
+| Chart introduces a property for configuring a new feature with a reasonable default value | minor version |
+| Chart renames an existing configuration property | major version |
+| Chart removes an existing configuration property | major version |
+
```
Comment (axdotl) [inline homepage/contribute.md]: I think it is worth to mention, that changing only the README.md also needs a patch version increase.
As this often is confusing to users.

**Comment by sophokles73** [comment]
@axdotl @monotek WDYT?

**Comment by monotek** [comment]
LGTM

**Comment by sophokles73** [comment]
@ctron @calohmn any thoughts?

**Pull request merged**